authors/apps/authentication/tests/test_mail.py

```diff
@@ -0,0 +1,31 @@
+"""
+this file contains tests to see whether an email is sent
+upon successful registration
+"""
+from rest_framework.test import APITestCase, APIClient
+from rest_framework.views import status
+from django.core import mail
+
+
+class EmailRegistrationTestCase(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.user_1 = {
+            "user": {
+                "email": "newmember@gmail.com",
+                "username": "New Member",
+                "password": "newmember000"
+            }
+        }
+
+    def test_email_notifications(self):
+        """
+        test if email is recieved on successful
+        registration
+        """
+        outbox_length = len(mail.outbox)
```

Comment:
:clap: This is a pretty cool test, I didn't know we can test mail sending by looking at the outbox.